### PR TITLE
Remove metering ConfigMap dependency.

### PR DESCRIPTION
**Pull request by yunkunrao**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
This PR removed metering ConfigMap dependency.

### CODE REVIEW

**Comment by yunkunrao** [comment]
This PR have a little ut coverage drop because metering delete some functions. At the same time, ut coverage include some other pkgs which aren't in the meteing scope. @LinuxSuRen

File: pkg/simple/client/metering/options.go [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
+	// cpu cost with above currency unit for per core per hour
+	MemPerGigabytesPerHour float64 `json:"memPerGigabytesPerHour" yaml:"memPerGigabytesPerHour"`
+	// mem cost with above currency unit for per GB per hour
+	IngressNetworkTrafficPerMegabytesPerHour float64 `json:"ingressNetworkTrafficPerMegabytesPerHour" yaml:"ingressNetworkTrafficPerGiagabytesPerHour"`
+	// ingress network traffic cost with above currency unit for per MB per hour
+	EgressNetworkTrafficPerMegabytesPerHour float64 `json:"egressNetworkTrafficPerMegabytesPerHour" yaml:"egressNetworkTrafficPerGigabytesPerHour"`
+	// egress network traffice cost with above currency unit for per MB per hour
+	PvcPerGigabytesPerHour float64 `json:"pvcPerGigabytesPerHour" yaml:"pvcPerGigabytesPerHour"`
+	// pvc cost with above currency unit for per GB per hour
+	CurrencyUnit string `json:"currencyUnit" yaml:"currencyUnit"`
```
Comment (LinuxSuRen) [inline pkg/simple/client/metering/options.go:15]: Normally the comments are above from the code lines. Please see also the introduction from [Golang blog](https://blog.golang.org/godoc).

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
I left some comments.

**Pull request merged**